### PR TITLE
Add support for range types

### PR DESCRIPTION
**Pull request by bcardarella**
* Added failing tests

### CODE REVIEW

**Review by carson-katri — APPROVED** [review]
LGTM

File: test/live_view_native/swiftui/rules_parser_test.exs [hunk 1]
```diff
@@ -189,6 +189,91 @@ defmodule LiveViewNative.SwiftUI.RulesParserTest do
...
+
+      assert parse(input) == output
+
+      input = "foo(..<10)"
+      output = {:foo, [], [{:"..<", [], [nil, 10]}]}
+
+      assert parse(input) == output
+
+      input = "foo(..<\"z\")"
+      output = {:foo, [], [{:..., [], [nil, "z"]}]}
```
Comment (NduatiK) [inline test/live_view_native/swiftui/rules_parser_test.exs:273]: Should be `:..<` not `:....`

**Review by NduatiK — CHANGES_REQUESTED** [review]

File: test/live_view_native/swiftui/rules_parser_test.exs (hunk 1 above)
Comment (bcardarella) [reply test/live_view_native/swiftui/rules_parser_test.exs:273]: yes, correcting

**Review by NduatiK — APPROVED** [review]
LGTM

**Pull request merged**